lib/mspec/runner/formatters/dotted.rb

```diff
@@ -25,7 +29,10 @@ def initialize(out=nil)
   def register
     (@timer = TimerAction.new).register
     (@tally = TallyAction.new).register
-    LeakCheckerAction.new.register if ENV['CHECK_LEAKS']
+    if ENV['CHECK_LEAKS']
+      LeakCheckerAction.new.register
+      ConstantsLeakCheckerAction.new.register
+    end
```

Comment:
I think a short comment with examples of shell commands (and `CHECK_LEAKS` values) and description would be helpful here or in the `constants_leak_checker.rb` file